readme.md

```diff
@@ -261,18 +261,18 @@ In general recommended VPS are those that are KVM/non-virtualized as Docker shou
 ### How to get Nextcloud running using the ACME DNS-challenge?
 You can install AIO in reverse proxy mode where is also documented how to get it running using the ACME DNS-challenge for getting a valid certificate for AIO. See the [reverse proxy documentation](./reverse-proxy.md). (Meant is the `Caddy with ACME DNS-challenge` section). Also see https://github.com/dani-garcia/vaultwarden/wiki/Running-a-private-vaultwarden-instance-with-Let%27s-Encrypt-certs#getting-a-custom-caddy-build for additional docs on this topic.
 
-### How to run Nextcloud locally?
-If you do not want to open Nextcloud to the public internet, you may have a look at the following documentation how to set it up locally: [local-instance.md](./local-instance.md)
+### How to run Nextcloud locally?  No domain wanted, or wanting intranet access within your LAN.
+If you do not want to open Nextcloud to the public internet, you may have a look at the following documentation on how to set it up locally: [local-instance.md](./local-instance.md), but keep in mind you're still required to have https working properly.  tldr: This is more of a temporary, internal testing move, which still requires additional, advanced configuration on your part.  Good luck!
```

Comment:
> tldr: This is more of a temporary, internal testing move, which still requires additional, advanced configuration on your part.  Good luck!

I am not sure about this wording. It is actually not an internal testing move but rather the recommended way how to set it up properly. Also the good luck wording is a bit weird imho

Reply:
Edited per your suggestions.